wideq/ac.py

```diff
@@ -173,18 +173,6 @@ def get_volume(self):
         value = self._get_control('SpkVolume')
         return int(value)
 
```

Comment:
This was moved to the base `Device` class since the implementation is identical with an ac, dryer or washer device.

Reply:
Good call!